BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/ConfirmEmailFragment.java

```diff
@@ -0,0 +1,52 @@
+package com.blueprint.foe.beetracker;
+
+import android.app.Fragment;
+import android.app.FragmentManager;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+public class ConfirmEmailFragment extends Fragment {
+    private static final String TAG = SignUpFragment.class.toString();
+
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.confirm_email_fragment, container, false);
+
+        TextView almostThere = (TextView) view.findViewById(R.id.almostThereText);
+        TextView confirmEmailDescription = (TextView) view.findViewById(R.id.confirmEmailDescription);
+
+        SharedPreferences sharedPref = getActivity().getSharedPreferences(getString(R.string.preference_file_key), Context.MODE_PRIVATE);
+        String email = sharedPref.getString(getString(R.string.signup_email), null);
+        String fullName = sharedPref.getString(getString(R.string.signup_name), null);
+
+        if (fullName != null && !fullName.isEmpty()) {
+            String[] nameArray = fullName.split(" ");
+            almostThere.setText(getString(R.string.almost_there, nameArray[0]));
+        }
+        if (email != null) {
+            confirmEmailDescription.setText(getString(R.string.confirm_email_description, email));
+        }
+
+        sharedPref.edit().remove(getString(R.string.signup_name)).commit();
```

Comment:
There is another screen (HomePage) where having the name is useful, I would keep it unless you have another reason to remove it.

Reply:
Is it just the name? Can I remove the email and password?

Reply:
Yeah just the name.

Reply:
Actually, wouldn't we be getting the name from server after logging in? This is just the name used during sign up, which shouldn't be used on the home page since different people can log in to the same device.